ElCoop/valuestore/src/Valuestore.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace valuestore;
+
+
+class Valuestore {
+
+	private $fileName;
+
+	public function __construct(string $fileName) {
+		if (file_exists($fileName)) {
+			if ((substr($fileName, -5) === '.json')) {
+				return $this->fileName = $fileName;
+			}
+			throw new \Exception('El-Coop Valuestore: File is not json');
```

Comment:
I don't think you need to write El-Coop Valuestore at start of the errore message - errors say where they where thrown from